tools/tdbg/tdbg_commands.go

```diff
@@ -198,6 +198,26 @@ func newAdminWorkflowCommands(clientFactory ClientFactory, prompterFactory Promp
 				return AdminRefreshWorkflowTasks(c, clientFactory)
 			},
 		},
+		{
+			Name:    "unblock",
```

Comment:
I won't call this unblock as it's too ambiguous what "unblock" cloud mean here. Workflow could be blocked waiting for a activity to be started, a child workflow to completed etc. 
I think it makes sense to make the name specific here and make it clear that it's scheduling a workflow task regardless of delays.